{"name": "test_add[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3", "trace": "self = <calctest.calc_case.calccase.TestCalcutor object at 0x000001542B7ACA30>\nget_calc = <calc_class.calcclass.Calculator object at 0x000001542B786CA0>\nget_add_datas = [1, 1, 3]\n\n    @allure.story(\"测试加法\")\n    @pytest.mark.run(order=1)#order控制方法执行顺序\n    def test_add(self,get_calc, get_add_datas):#定义测试加法案例方法，实例化加法\n        # 步骤说明\n        with allure.step(\"加法计算\"):\n            result = get_calc.add(get_add_datas[0],get_add_datas[1])#获取测试数据\n        if isinstance(result, float):#判断数据类型，如果是浮点数，强制保留小数点后两位\n            result = round(result, 2)\n>       assert result == get_add_datas[2]#断言:判断是否相等，相等则pass，不相等则fail\nE       assert 2 == 3\n\ncalc_case\\calccase.py:18: AssertionError"}, "steps": [{"name": "加法计算", "status": "passed", "start": 1612344421000, "stop": 1612344421000}], "parameters": [{"name": "get_add_datas", "value": "[1, 1, 3]"}], "start": 1612344421000, "stop": 1612344421000, "uuid": "2e11d605-044a-4af9-9de2-8af22c280c1e", "historyId": "2b1746b828adf3bb52f30277149d68e9", "testCaseId": "166595ee5e0d6ded0a0a8ccd5f5a9ec9", "fullName": "calc_case.calccase.TestCalcutor#test_add", "labels": [{"name": "story", "value": "测试加法"}, {"name": "feature", "value": "计算器测试"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "calc_case"}, {"name": "suite", "value": "calccase"}, {"name": "subSuite", "value": "TestCalcutor"}, {"name": "host", "value": "SC-202005180944"}, {"name": "thread", "value": "19060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "calc_case.calccase"}]}